src/node_file.cc

```diff
@@ -2626,6 +2626,32 @@ static void FChown(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void FChownSync(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  const int argc = args.Length();
+  CHECK_GE(argc, 3);
+
+  CHECK(args[0]->IsInt32());
+  const uv_file fd = static_cast<uv_file>(args[0].As<Int32>()->Value());
+
+  CHECK(args[1]->IsInt32());
+  const uv_uid_t uid = static_cast<uv_uid_t>(args[1].As<Int32>()->Value());
+
+  CHECK(args[2]->IsInt32());
+  const uv_gid_t gid = static_cast<uv_gid_t>(args[2].As<Int32>()->Value());
+
+  uv_fs_t req;
+  auto cleanup = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
+  FS_SYNC_TRACE_BEGIN(fchown);
+  int err = uv_fs_fchown(nullptr, &req, fd, uid, gid, nullptr);
+  FS_SYNC_TRACE_END(fchown);
+  if (err < 0) {
+    return env->ThrowUVException(err, "fchown");
```

Comment:
This does not need a new binding, it can just be done by switching `SyncCall` to `SyncCallAndThrowOnError` in the original implementation.